src/ext/components/partial-dependence-input.pug

```diff
@@ -32,6 +32,11 @@
         td
           input.flow-textbox(type='text' data-bind="value:nbins")
         td How many levels should PDP compute. More levels will make it slower.
+      tr
+        th target
+        td
+          input.flow-textbox(type='text' data-bind="value:targets")
```

Comment:
The value need to be `targets` not `target` due to parameter name in Java backend where multiple target classes are available.